src/util.jl

```diff
@@ -1,10 +1,21 @@
+# cached queues
+const DEFAULT_QUEUES = Dict{Context, CmdQueue}()
```

Comment:
This will prevent garbage collection for Contexts, so this should probably be a `WeakKeyDict`